tools/cxbe/Main.cpp

```diff
@@ -132,7 +178,7 @@ int main(int argc, char *argv[])
             LogoPtr = &logo;
         }
 
-        Xbe *XbeFile = new Xbe(ExeFile, szXbeTitle, dwTitleId, bRetail, LogoPtr, szDebugPath);
+        Xbe *XbeFile = new Xbe(ExeFile, szXbeTitle, dwTitleId, dwRegions, bRetail, LogoPtr, szDebugPath);
```

Comment:
This does violate the clang-format rules (as do some of the changes carried over from the last commit).